website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx

```diff
@@ -97,18 +97,18 @@ Map that contains the details about the gateway policy. The `apiVersion`, `kind`
 
 ### `spec.idleTimeout
 
-Specifies the total amount of time permitted for the request stream to be idle.
+Specifies the total amount of time permitted for the request stream to be idle. Must specify a parseable number and a unit, for example "5s". 
```

Comment:
Because this uses the Go `time` package under the hood, it might be good to reference that: https://pkg.go.dev/maze.io/x/duration#ParseDuration

Reply:
Not that it negates your point, but its actually using [metav1.Duration](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#Duration) in consul-k8s, which I think behaves slightly differently (which is actually why we need this update to begin with.) 

I'll defer to @trujillo-adam on phrasing here though.